repository/SmalltalkCI-Core.package/SCIMetacelloLoadSpec.class/instance/requiresLatestMetacello.st

```diff
@@ -0,0 +1,11 @@
+private
+requiresLatestMetacello
+	"Check for missing features."
+
+	"Credentials parameters (https://github.com/Metacello/metacello/pull/536)"
```

Comment:
I wonder: should smalltalkCI print something to stdout informing the user why it has decided to upgrade Metacello? Something like:
> Upgrading to the latest Metacello because credentials are provided...

Reply:
Good idea! I will implement this soon.